docs/guides/abn-tests.mdx

```diff
@@ -1,49 +1,44 @@
+### [You don't need large sample sizes to run A/B Tests](https://blog.statsig.com/you-dont-need-large-sample-sizes-to-run-a-b-tests-6044823e9992)
 
+:::
 
 ## Step 1 - Create a new experiment in the Statsig console
 
-Log into the Statsig console at https://console.statsig.com/ and navigate to Experiments+ in the left-hand navigation panel.
+Log into the Statsig console at https://console.statsig.com/ and navigate to Experiments in the left-hand navigation panel.
 
 Click on the **Create** button and enter the name and description for your experiment. Click **Create**.
 
-![image](https://user-images.githubusercontent.com/1315028/138970611-1e46fe5a-6119-4be3-8cc1-0b296477d6c8.png)
-
-In the **Allocation** panel, enter the percentage of users that you want to allocate to this experiment. By default, each experiment runs in its own layer. When you want to create an experiment that excludes any users exposed to other experiments, you will also want to click the checkbox for **Add Layer**. As this is our first experiment, enabling this isolation isn't relevant for us at this stage.
-
-![image](https://user-images.githubusercontent.com/1315028/138972419-b7c42f97-29ec-407e-851f-3301130a21c5.png)
+![image](https://user-images.githubusercontent.com/1315028/154571621-4022dddb-6689-4a25-9401-f813a159ef9f.png)
 
-Staying in the experiment **Setup** tab, under the **Variations** panel, enter the variant names and click on **Add Another Variant** to enter more variants. You'll find that the split percentages between these variants automatically change to evenly distribute users between these variants.
+In the Setup tab, define the test groups for this experiment. You can define what percentage of users should receive the group's configured values. Click on the pencil icon next to the _sample_parameter_ and enter the name and type of your experiment parameter, and click **Confirm**. Enter the values that the parameter will take for the control and test groups.  
 
-![image](https://user-images.githubusercontent.com/1315028/139104398-bf46b591-e3e7-4c6a-bde1-508898229660.png)
+![image](https://user-images.githubusercontent.com/1315028/154576194-0b38bbe6-feb8-47a1-b460-3c0c2e9e58c6.png)
 
-Staying in the **Variations** panel, click on the pencil icon next the _sample_parameter_ and enter the name of your experiment parameter and the type, and click **Confirm**
-
-![image](https://user-images.githubusercontent.com/1315028/138972604-3bb43962-6991-4eac-9fba-a9ab165ea1d2.png)
+In the **Allocation** panel, enter the percentage of users that you want to allocate to this experiment. By default, each experiment runs in its own layer. When you want to create an experiment that excludes any users exposed to other experiments, you will also want to click the checkbox for **Add Layer**. As this is our first experiment, enabling this isolation isn't relevant for us at this stage.
```

Comment:
Its getting into the details of layers going to add unnecessary confusion to a first time experiment setup?